### PR TITLE
feat: show quorum progress

### PR DESCRIPTION
**Pull request by wa0x6e**
### Summary

Closes: #279 

Show quorum progress below proposal results, and in proposalListItem

Screenshot 2024-04-16 at 14 00 17

Screenshot 2024-04-16 at 02 23 26

Screenshot 2024-04-16 at 02 27 55

On quorum of reject, the result progress bar only show progress of the rejection choice

Screenshot 2024-04-16 at 14 00 50

NOTE: on active shutter proposals, the progress of all proposals using quorum of rejection will always be 0%, as choices are hidden until proposal is closed.

### How to test

1. You can see some tests proposals with default and rejection quorum on http://localhost:8080/#/s:test.wa0x6e.eth
2. You can also create some proposals with default and rejection quorum, and see the progress after voting

### CODE REVIEW

**Comment by bonustrack** [comment]
On the feed we shouldnt have uppercase or text skin link color

**Comment by bonustrack** [comment]
- On the feed quorum should appear after votes (not in last position)
- Do you cap quorum at 100%? Not sure we should do this

**Comment by bonustrack** [comment]
Did you tested on SX spaces?

**Comment by wa0x6e** [comment]
On sx spaces, the `quorum` field from the graphql proposal query is always returning 0

**Review by bonustrack — APPROVED** [review]
tACK

File: apps/ui/src/helpers/quorum.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Proposal } from '@/types';
+
+const REJECTION_QUORUM_CHOICE_INDEX = 1;
+
+export function quorumProgress(proposal: Proposal): number {
+  const totalScore =
+    proposal.quorum_type === 'rejection'
+      ? proposal.scores
+          .filter((c, i) => i === REJECTION_QUORUM_CHOICE_INDEX)
+          .reduce((a, b) => a + b, 0)
```
Comment (Sekhmet) [inline apps/ui/src/helpers/quorum.ts:10]: Would it be the same? Looks a bit overcomplex by doing it on arrays with filter/reduce.

```suggestion
      ? proposal.scores[REJECTION_QUORUM_CHOICE_INDEX] ?? 0
```

**Review by Sekhmet — APPROVED** [review]
tACK

**Pull request merged**